src/App.tsx

```diff
@@ -9,16 +10,68 @@ const goodsFromServer: string[] = [
   'Apple',
   'Bread',
   'Fish',
-  'Honey',
   'Jam',
+  'Honey',
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component<Props, {}> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {selectedGood}
+          { ' ' }
+          is selected
```

Comment:
In case no good is selected, this msg will be "No goods is selected", it must be "No goods selected" - it's grammatically correct.